packages/lib/link/messages.js

```diff
@@ -916,6 +916,22 @@ messages.debugWsMessage = new Event({
 	},
 });
 
+messages.liveUpdateSlaves = new Event({
+	type: "live_update_slaves",
+	links: ["master-control"],
+	eventProperties: {
+		item: {
```

Comment:
The convention in this file is that properties are put in `"quotes"` when they are present in the payload, and are not quoted when they are JSON Schema keywords. So item should be quoted here.